common-items.lic

```diff
@@ -372,21 +406,15 @@ module DRCI
     case DRC.bput("open #{container}", $OPEN_CONTAINER_SUCCESS_PATTERNS, $OPEN_CONTAINER_FAILURE_PATTERNS)
     when *$OPEN_CONTAINER_SUCCESS_PATTERNS
       return true
-    when *$OPEN_CONTAINER_FAILURE_PATTERNS
-      return false
-    else
-      return false
     end
+    return false
```

Comment:
Six of one, half dozen of another... this change removes unecessary code at this time but functionally same result.